libnavigation-base/src/main/java/com/mapbox/navigation/base/route/dto/BannerComponentsNavigationDto.kt

```diff
@@ -0,0 +1,27 @@
+package com.mapbox.navigation.base.route.dto
+
+import com.google.gson.annotations.SerializedName
+import com.mapbox.navigation.base.route.model.BannerComponentsNavigation
+
+class BannerComponentsNavigationDto(
```

Comment:
Why do we need models and DTOs? Shouldn't they be the same? It seems we’re duplicating a lot of code unnecessarily.

Reply:
Under [discussion](https://github.com/mapbox/navigation-sdks/issues/206)